libp2p/pubsub/pubsub.py

```diff
@@ -128,7 +141,7 @@ def get_hello_packet(self) -> bytes:
                         continue
                     # TODO(mhchia): This will block this read_stream loop until all data are pushed.
                     #   Should investigate further if this is an issue.
-                    await self.push_msg(msg_forwarder=peer_id, msg=msg)
+                    asyncio.ensure_future(self.push_msg(msg_forwarder=peer_id, msg=msg))
```

Comment:
An issue(#193) was opened to track this but we should expand the coverage to whole repo instead of just pubsub/. Currently there are also a few such untracked tasks in kademlia/, network/ and stream_muxer/.